content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst

```diff
@@ -2,59 +2,70 @@
 
-Manage your production planning 
-=================================
+Manage production planning
+==========================
 
-There are several methods to manage the triggering of the various manufacturing orders. 
+There are several methods to manage manufacturing order automation for products with multilevel
+:abbr:`BoMs (Bills of Materials)`.
 
-If every time a manufacturing order is confirmed for the main product, you'd like one for 
-the semi-finished products as well, you have two options: 
+.. note:: 
+    Semi-finished products are specifically used to manage manufacturable products with multilevel
+    BoMs. If a BoM is being created simply to organize components or bundle sellable products,
+    using :doc:`Kits <kit_shipping>` is the more appropriate option.
 
-Option 1 : Create re-ordering rules for the semi-finished products, with both the minimum 
-and maximum desired stock quantities at 0. 
+To automatically trigger manufacturing orders for semi-finished products after confirming a
+manufacturing order for the main product, there are two options:
 
-.. image:: sub_assemblies/sf_3.png
-    :align: center
+**Option 1 (recommended):** Create *Reordering Rules* for the semi-finished products and set both
+the minimum and maximum desired stock quantities to '0'. To review how to set up reordering rules,
+refer to :doc:`this page <../../../inventory_and_mrp/purchase/products/reordering>`.
 
-Option 2 : Use the Replenish on Order (MTO) route on the semi-finished product, as well as 
-the manufacturing one. 
+**Option 2:** Activate the *Replenish on Order (MTO)* and *Manufacture* routes under the
+:guilabel:`Inventory` tab of the semi-finished product’s product form.
 
-Note that Option 1 is usually recommended over Option 2 as it is more flexible. The MTO route 
-creates a unique link between the semi-finished and the top level product, whereas the 
-re-ordering rule allows you to simply unreserve that production from the top level product
-production and redirect it to another, more pressing demand, for example. 
+Option 1 is more flexible than Option 2 and is therefore recommended. Reordering rules do not
+directly link demand to replenishment, and therefore allow stocks to be unreserved and redirected
+to other orders, if necessary. The Replenish on Order (MTO) route creates a unique link between the
+semi-finished and top-level products, exclusively reserving quantities for the confirmed top-level
+manufacturing order.
 
-In any case, as soon as the semi-finished product is produced, it will become
-available in the manufacturing order of the final product, as shown below.
+Regardless of the method chosen, semi-finished products must be fully manufactured before
+manufacturing can begin on the top-level product.
 
-.. image:: sub_assemblies/sf_4.png
-    :align: center
\ No newline at end of file
+.. image:: sub_assemblies/semifinished_on_MO.png
```

Comment:
Replace underscores with hyphens.
As well, [media file names can not have capital letters](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#media-files).
```suggestion
.. image:: sub_assemblies/semifinished-on-mo.png
```